openlibrary/templates/admin/attach_debugger.html

```diff
@@ -1,10 +1,10 @@
-$def with (keys="", error="")
+$def with (python_version, keys="", error="")
 
 $var title: Attach Debugger
 
 <div id="contentHead">
     $:render_template("admin/menu")
-    <h1>Attach Debugger</h1>
+    <h1>Attach Debugger on Python $:python_version</h1>
```

Comment:
The colon is only needed if you want to avoid HTML escaping. And also to extract nutrients/water from your food during digestion :+1:

```suggestion
    <h1>Attach Debugger on Python $python_version</h1>
```